removesDup.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+
+int main(){
+	int n; // number of elements in array
+	int a[n]; // creates an array
```

Comment:
Once I fixed the seg fault, your code mostly worked - there were a few logical errors in terms of the number of elements accepted for the given array size, but overall it worked okay!